src/libpmemfile-core/file.c

```diff
@@ -1674,3 +1674,171 @@ pmemfile_truncate(PMEMfilepool *pfp, const char *path, off_t length)
 
 	return 0;
 }
+
+static int
+vinode_chown(PMEMfilepool *pfp, struct pmemfile_cred *cred,
+		struct pmemfile_vinode *vinode, uid_t owner, gid_t group)
+{
+	struct pmemfile_inode *inode = vinode->inode;
+	int error = 0;
+
+	if (owner == (uid_t)-1 && group == (gid_t)-1)
+		return 0;
+
+	os_rwlock_wrlock(&vinode->rwlock);
+
+	if (!(cred->caps & (1 << PMEMFILE_CAP_CHOWN))) {
```

Comment:
Well, not that it matters a lot, but this section of code doesn't seem to use the vinode either...
So why lock the vinode before it?